.github/workflows/check-archs-consistency.yml

```diff
@@ -0,0 +1,26 @@
+name: Test consistency between Paper & Fabric
+on: 
+  pull_request:
+    branches:
+      - main
+    paths: 
+      - src/fabric/**
+      - android/src/paper/java/com/facebook/react/viewmanagers/**
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: check-archs-consistency-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js 18
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - name: Install node dependencies
+        run: yarn
+      - name: Check Android Paper & Fabric generated interfaces consistency
+        run: yarn check-archs-consistency
```

Comment:
no newline

Reply:
fixed in: https://github.com/software-mansion/react-native-svg/pull/2357/commits/b20a8c8157363c41523bf15b1ff480bc0ad133e3